### PR TITLE
Announce Lagom 1.3.6

### PR DESCRIPTION
**Pull request by TimMoore**

### CODE REVIEW

File: src/blog/lagom-1-3-6.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    The Lagom team has released Lagom 1.3.6
+---
+
+Following from the [first preview release of Lagom 1.4.0](lagom-1-4-0-M1.md), several smaller improvements and fixes that were made since then have been backported to the 1.3 stable branch.
+
+1.3.6 contains small bug fixes and minor enhancements, and a couple of APIs were deprecated to fix typos and inconsistencies. One important fix improves the reliability of publishing events to Kafka by ensuring that topic producers are always restarted after failure (with exponential back off).
+
+We've continued to get great pull requests from the Lagom community. Thanks to GitHub users [Edouard Kaiser](https://github.com/edouardKaiser), [AmeerGN](https://github.com/AmeerGN), [Elijah Rippeth](https://github.com/erip), [mihbor](https://github.com/mihbor), and Outreachy intern [Yuliana Apaza](https://github.com/yg-apaza) for your contributions to Lagom 1.3.6!
+
+## Updating
```
Comment (rstento) [inline src/blog/lagom-1-3-6.md]: ## Updating a project to use the new release.

File: src/blog/lagom-1-3-6.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Lagom 1.3.6 is released!
+date: 2017-07-17
+author_github: TimMoore
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.3.6
+---
+
```
Comment (rstento) [inline src/blog/lagom-1-3-6.md]: I would re-structure the first two paragraphs into a bulleted list, something like:

For the specific changes in Lagom 1.3.6, see (link to either the changelog.html or the section below, see my comments below) As an overview, this release contains the following:
- Small improvements from the preview release of Lagom 1.4.0, which have been backported to this release.
- Small bug fixes and minor enhancements 
- Deprecated APIs to fix typos and inconsistencies.
- Improvement in event publishing to Kafka by ensuring that topic producers are always restarted after failure (with exponential back off) (And, I would add whether it happens without the developer having to do anything)

File: src/blog/lagom-1-3-6.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+We've continued to get great pull requests from the Lagom community. Thanks to GitHub users [Edouard Kaiser](https://github.com/edouardKaiser), [AmeerGN](https://github.com/AmeerGN), [Elijah Rippeth](https://github.com/erip), [mihbor](https://github.com/mihbor), and Outreachy intern [Yuliana Apaza](https://github.com/yg-apaza) for your contributions to Lagom 1.3.6!
+
+## Updating
+
+To update an sbt project, change the version of `lagom-sbt-plugin` in your `project/plugins.sbt` file.
+
+To update a Maven project, change the `lagom.version` property in your top-level `pom.xml`.
+
+## Change log
+
```
Comment (rstento) [inline src/blog/lagom-1-3-6.md]: Ah, here it is, I'd either move the link to changlog.html up into the bulleted list, or link to this section. I'm not sure on what the difference is between the link and the information in this section?

File: src/blog/lagom-1-3-6.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Following from the [first preview release of Lagom 1.4.0](lagom-1-4-0-M1.md), several smaller improvements and fixes that were made since then have been backported to the 1.3 stable branch.
+
+1.3.6 contains small bug fixes and minor enhancements, and a couple of APIs were deprecated to fix typos and inconsistencies. One important fix improves the reliability of publishing events to Kafka by ensuring that topic producers are always restarted after failure (with exponential back off).
+
+We've continued to get great pull requests from the Lagom community. Thanks to GitHub users [Edouard Kaiser](https://github.com/edouardKaiser), [AmeerGN](https://github.com/AmeerGN), [Elijah Rippeth](https://github.com/erip), [mihbor](https://github.com/mihbor), and Outreachy intern [Yuliana Apaza](https://github.com/yg-apaza) for your contributions to Lagom 1.3.6!
+
+## Updating
+
+To update an sbt project, change the version of `lagom-sbt-plugin` in your `project/plugins.sbt` file.
```
Comment (rstento) [inline src/blog/lagom-1-3-6.md]: So, the only thing they need to do is change the version number? What about search for and replace any deprecated APIs they were using?

**Comment by TimMoore** [comment]
Bummer.. this was merged before I got your comments, @rstento. I'll publish as is so we don't get delayed for another day, then I'll open a new pull request with the changes you suggested.

**Comment by TimMoore** [comment]
See #87 for updates

**Pull request merged**